modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -133,6 +133,16 @@
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
 	},
+	{
+		"classNames": [
+			"CommerceAddressLocalService",
+			"CommerceAddressLocalServiceUtil"
+		],
+		"from": "updateCommerceAddress(CommerceAddress)",
+		"issueKey": "LPS-186925",
```

Comment:
@MicaelleSilvaa, `issueKey` incorrectly sorted. Is it related to a branch rebase?